repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc

```diff
@@ -0,0 +1,133 @@
+= OpenTracing Service
+
+The OpenTracing service (http://opentracing.io/) is a service in *Repose* which allows for sending tracing data to a Tracing service.
+
+The initial release of the OpenTracing service only supports using the Jaeger (http://jaeger.readthedocs.io/en/latest/) Tracer.
+However, other Tracers can be integrated in future OpenTracing service updates.
+By default, the service is disabled, but it can be enabled by including an `open-tracing.cfg.xml` configuration.
```

Comment:
Do we want to make a note that the service cannot be disabled once it has been enabled unless Repose is restarted?

Reply:
Added.